CONTRIBUTING.md

```diff
@@ -9,6 +9,11 @@ Others are specific conventions chosen for the pySDC library, that may follow Py
 Finally, while `pySDC` provides many base functionalities that implement classical flavors of SDC, it also allows problem-specific applications through Object-Oriented Programming (OOP) and the implementation of custom inherited classes.
 This follows a specific OOP framework, you can look at the page on [custom implementations](./docs/contrib/04_custom_implementations.md) for more details. Additional guideline are also given on how to [document the code](./docs/contrib/05_documenting_code.md) in `pySDC`.
 
+> 📣 Some core components (collocation matrix, $Q-\Delta$ coefficients, Lagrange barycentric interpolation) 
+> are currently implemented and developed in the [`qmat` companion package](https://github.com/Parallel-in-Time/qmat).
```

Comment:
I'd leave out the `currently`, because this sounds like this is a temporary thing.

Reply:
Done